File: exs/mundo_1/c/013.c
/*
    013 - Reajuste salarial
        Faça um algoritmo que leia o salário de um funcionário e mostre seu novo
        salário, com 15% de aumento
*/

#include <stdio.h>

int main(void)
{
    float sal_inicial;
    printf("Digite o salário para calcular o aumento de 15%%: R$");
    scanf("%f", &sal_inicial);

    printf("Salário com aumento: R$%.2f\n", sal_inicial * 1.15);
    return 0;
}

File: exs/mundo_1/rust/013.rs
/*
Desafio 013

Problema: Faça um algoritmo que leia o salário de um funcionário
          e mostre seu novo salário, com 15% de aumento.
*/

fn main() {
    let mut salario = String::new();

    println!("Informe seu sálario (R$):");
    std::io::stdin().read_line(&mut salario).unwrap();

    let salario: f32 = salario.trim().parse().expect("Erro ao ler sálario");

    let salario_aumentado = salario * 1.15;

    println!("Salário final: R${salario_aumentado:.2}");
}
